cves/kernel/CVE-2013-3228.yml

```diff
@@ -1,261 +1,104 @@
-  note:
-  answer:
-subsystem:
-  question: |
-    What subsystems was the mistake in? These are WITHIN linux kernel
-
-    Determining the subsystem is a subjective task. This is to help us group
-     similar vulnerabilities, so choose a subsystem that other vulnerabilities would be in. Y
-
-    Some areas to look for pertinent information:
-      - Bug labels
-      - Directory names
-      - How developers refer to an area of the system in comments,
-        commit messages, etc.
-
-    Look at the path of the source code files code that were fixed to get
-    directory names. Look at comments in the code. Look at the bug reports how
-    the bug report was tagged.
-
-    Example linux kernel subsystems are:
-      * drivers
-      * crypto
-      * fs
-      * net
-      * lib
-
-    Name should be:
-      * all lowercase English letters
-      * NOT a specific file
-      * can have digits, and _-@/
-
-    Can be multiple subsystems involved, in which case you can make it an array
-    e.g.
-        name: ["subsystemA", "subsystemB"] # ok
-        name: subsystemA # also ok
-  name:
-  note:
-interesting_commits:
-  question: |
-    Are there any interesting commits between your VCC(s) and fix(es)?
-
-    Use this to specify any commits you think are notable in some way, and
-    explain why in the note.
-
-    Example interesting commits:
-      * Mentioned as a problematic commit in the past
-        e.g. "This fixes regression in commit xys"
-      * A significant rewrite in the git history
-      * Other commits that fixed a similar issue as this vulnerability
-      * Anything else you find interesting.
-  commits:
-  - commit:
+    If you think of another lesson we covered in class that applies here, feel
+    free to give it a small name and add one in the same format as these.
+  serial_killer:
     note:
-  - commit:
+    applies:
+  complex_inputs:
     note:
-i18n:
-  question: |
-    Was the feature impacted by this vulnerability about internationalization
-    (i18n)?
-
-    An internationalization feature is one that enables people from all
-    over the world to use the system. This includes translations, locales,
-    typography, unicode, or various other features.
-
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+    applies:The msg parameter contains a large number of attributes that may not be initialized. Needing to handle these to prevent an information leak was unclear initially.
+  distrust_input:
+    note:
+    applies:
+  least_privilege:
+    note:
+    applies:
+  native_wrappers:
+    note:
+    applies:
+  defense_in_depth:
+    note:
+    applies:
+  secure_by_default:
+    note:
+    applies:
+  environment_variables:
+    note:
+    applies:
+  security_by_obscurity:
+    note:
+    applies:
+  frameworks_are_optional:
+    note:
+    applies:
+reviews: []
 sandbox:
+  note:
+  answer:
```

Comment:
I found this yml parse error by presing "." on GitHub and opening this in GitHub.dev - the YML parser there caught it.